dev-libs/amdgpu-pro-opencl/amdgpu-pro-opencl-18.20.606296-r1.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MULTILIB_COMPAT=( abi_x86_{32,64} )
+
+inherit unpacker multilib-minimal
+
+SUPER_PN='amdgpu-pro'
+MY_PV=$(ver_rs 2 '-')
+
+DESCRIPTION="Proprietary OpenCL implementation for AMD GPUs"
+HOMEPAGE="https://support.amd.com/en-us/kb-articles/Pages/Radeon-Software-for-Linux-Release-Notes.aspx"
+SRC_URI="${SUPER_PN}-${MY_PV}.tar.xz"
+
+LICENSE="AMD-GPU-PRO-EULA"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND=""
```

Comment:
Empty variable can be omitted.

Reply:
I usually keep empty vars for clarity. But maybe it's time to adjust my habits.

btw, did you ever think of getting quizzes done and becoming a dev xD?

Reply:
@gyakovlev I don't know if it's standardized in Gentoo, but I was told they can be removed in newer EAPI :)

Yes I did, but haven't got it yet ;)